src/hooks/useSaveShortcut.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
+
+const useSaveShortcut = (saveCallback: () => Promise<void>) => {
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if ((event.ctrlKey || event.metaKey) && event.key === 's') {
```

Comment:
Ctrl+S will also work on macOS. Detecting the platform would require touching the main process, so for now, I’ve opted to leave it as is. We can revisit this when our keybinding system expands and needs more granular control.